cedar-policy-validator/src/human_schema/err.rs

```diff
@@ -415,17 +415,55 @@ impl Diagnostic for ToJsonSchemaError {
     }
 }
 
+/// Warnings when builtin Cedar names are shadowed
+#[derive(Debug, Clone, Error)]
+#[error("The name `{name}` shadows a builtin Cedar name. You'll have to refer to the builtin as `__cedar::{name}`.")]
+pub struct ShadowsBuiltinWarning {
```

Comment:
Style question: do we include `Warning` or `Error` in the struct names? I'd vote no, but mainly we should be consistent.

Reply:
I'm fine either ways as long as there are no suffixes in error variants.